doc/user-manual.md

```diff
@@ -0,0 +1,738 @@
+    - [Users - Super user, inactive user, never logged in audit](#users---super-user-inactive-user-never-logged-in-audit)
+    - [Users : Create users based on a csv](#users--create-users-based-on-a-csv)
+    - [Coordinates](#coordinates)
+    - [XLSForm - Generate a basic xlsform for a program](#xlsform---generate-a-basic-xlsform-for-a-program)
+    - [XLSForm - generate a xlsform from a DataSet](#xlsform---generate-a-xlsform-from-a-dataset)
+    - [XLSForm - Diff two xlsform](#xlsform---diff-two-xlsform)
+    - [Dataviz - update custom attributes of program indicator](#dataviz---update-custom-attributes-of-program-indicator)
+    - [Your recipe ?](#your-recipe-)
+  - [Specific recipes](#specific-recipes)
+- [Coding tutorial](#coding-tutorial)
+  - [Turn an api call in to a csv](#turn-an-api-call-in-to-a-csv)
+    - [v0.0 get all orgunits](#v00-get-all-orgunits)
+    - [v0.1 get all orgunits for a certain level as parameter](#v01-get-all-orgunits-for-a-certain-level-as-parameter)
+  - [Generate a csv to create users based on the level 3](#generate-a-csv-to-create-users-based-on-the-level-3)
+  - [Synchronous api calls in a loop](#synchronous-api-calls-in-a-loop)
+  - [Generate a json with download prompt](#generate-a-json-with-download-prompt)
+  - [Dhis2 periods](#dhis2-periods)
+  - [Let's update things](#lets-update-things)
+    - [First a readonly version](#first-a-readonly-version)
+    - [Then introduce a "dryRun" mode](#then-introduce-a-dryrun-mode)
+    - [Then test on the first record](#then-test-on-the-first-record)
+    - [Then run on all users](#then-run-on-all-users)
+    - [If the recipe is here to stay](#if-the-recipe-is-here-to-stay)
+  - [We want more](#we-want-more)
+
+
+# Disclaimer
+
+> When the only tool you have is TaskR, everything start to look like a recipe
+
+It gets addictive, you can spend hours automating something that could have been done in 10 minutes clicking in a few screens.
+
+> With great power comes great responsibility
+
+Taskr recipes runs with the user privilege. If you are admin the recipe has all the possibilities the admin has.
+
+> To err is human, but for a real disaster you need a computer"
+
+What can harm a dhis2 by clicking the wrong option in a menu in the dataElement option, can turn into a disaster when the same operation is repeated in a loop in taskr. It can get worst when what is in the loop is not the correct action or was not meant to run on all data elements. Be careful.
+
+
+# Small tour
+
+## Open the app
+
+Once the app installed you can locate the app in the menu.
+
+![recipes-list](./user-manual-taskr-app.jpg)
+
+## Recipes list
+
+![recipes-list](./user-manual-recipes-list.jpg)
+
+
+The recipes list screen is accessible via the top left hamburger menu icon. A list of recipes is shown :
+   - grey out recipes are "built in".
+   - blank recipees are the one created and saved in your dhis2
+
+Click on the run button, you arrive fill on the recipe page.
```

Comment:
This means it actually do now run it immediatly? Maybe change "run" to "show" on the button then?